yapenv/cli.py

```diff
@@ -30,9 +30,7 @@ def env_file(self) -> str:
         return self.get("env_file", os.environ.get("YAPENV_ENV_FILE", ".env"))
 
     def load(
-        self,
-        resolve_imports: bool = True,
-        ignore_environment: bool = False,
+        self, resolve_imports: bool = True, ignore_environment: bool = False, inherit_depth: int = None
```

Comment:
Kinda weird formatting, should be this right?
```suggestion
        self,
        resolve_imports: bool = True,
        ignore_environment: bool = False,
        inherit_depth: int = None
```

Reply:
Fixed.